extension.js

```diff
@@ -114,6 +114,20 @@ function overlay_key_changed(settings) {
     overlay_key_action = settings.get_enum("overlay-key-action");
 }
 
+function hide_application_menu(hide) {
+    if (hide) {
+	appMenu_signal_show = Main.panel.statusArea.appMenu.connect("show", function() {
+	    Main.panel.statusArea.appMenu.hide();
+	});
+	Main.panel.statusArea.appMenu.hide();
+    } else {
+	if (appMenu_signal_show != null) {
+	    Main.panel.statusArea.appMenu.disconnect(appMenu_signal_show);
+	    appMenu_signal_show = null;
+	}
+	Main.panel.statusArea.appMenu.show();
+    }
```

Comment:
Sorry for the style nitpick, but it looks like the indentation from #263 was lost in this PR. Could you correct that please?

Reply:
Yep, sorry about that... forgot to expand tabs... should be fixed now.